docs/sources/mimir/operators-guide/architecture/components/query-scheduler/index.md

```diff
@@ -50,14 +50,57 @@ To use the query-scheduler with DNS-based service discovery, configure the query
 
 > **Note:** The querier pulls queries only from the query-frontend or the query-scheduler, but not both. `-querier.frontend-address` and `-querier.scheduler-address` options are mutually exclusive, and only one option can be set.
 
-### Ring-based service discovery (experimental)
+### Ring-based service discovery
 
 To use the query-scheduler with ring-based service discovery, configure the query-schedulers to join their hash ring, and the query-frontends and queriers to discover query-scheduler instances via the ring:
 
 1. [Configure the hash ring]({{< relref "../../../configure/configure-hash-rings.md" >}}) for the query-scheduler.
 1. Set `-query-scheduler.service-discovery-mode=ring` (or its respective YAML configuration parameter) to query-scheduler, query-frontend and querier.
 1. Set the `-query-scheduler.ring.*` flags (or their respective YAML configuration parameters) to query-scheduler, query-frontend and querier.
 
+#### Migrate from DNS-based to ring-based service discovery
+
+To migrate the query-scheduler from [DNS-based service discovery](#dns-based-service-discovery) to [ring-based service discovery](#ring-based-service-discovery) you can follow these steps:
+
+1. Configure the **query-scheduler** instances to join a ring:
+   ```
+   -query-scheduler.service-discovery-mode=ring
+
+   # Configure the query-scheduler ring backend (e.g. "memberlist").
+   -query-scheduler.ring.store=<backend>
+
+   # If the configured <backend> is "memberlist", then ensure memberlist is configured for the query-scheduler.
```

Comment:
```suggestion
   # If the configured <backend> is `memberlist`, then ensure memberlist is configured for the query-scheduler.
```
Do you mean verbatim `memberlist`?

Reply:
Since we're already a inside code block using the backtick won't be formatted.